Billbee.Api.Client.Test/EndPointIntegrationTests/CustomerAddressesEndPointIntegrationTest.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Billbee.Api.Client.Model;
+using Billbee.Api.Client.Test.EndPointIntegrationTests.Helpers;
+
+namespace Billbee.Api.Client.Test.EndPointIntegrationTests.Helpers
+{
+    public static partial class TestData
+    {
+        public static CustomerAddress CustomerAddress =>
+            new CustomerAddress
+            {
+                FirstName = "John",
+                LastName = "Doe",
+                Street = "Mustergasse",
+                Housenumber = "1",
+                Zip = "12345",
+                City = "Musterstadt",
+                AddressType = 1,
+                CountryCode = "DE",
+                CustomerId = 0
+            };
+        
+        public static CustomerAddress GetCustomerAddress(long? customerId)
+        {
+            var address = CustomerAddress;
+            address.CustomerId = customerId;
+            return address;
+        }
+    }
+}
+
+namespace Billbee.Api.Client.Test.EndPointIntegrationTests
+{
+    [TestClass]
+    public class CustomerAddressesEndPointIntegrationTest
+    {
+#pragma warning disable CS8618
+        public TestContext TestContext { get; set; }
+#pragma warning restore CS8618
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            IntegrationTestHelpers.CheckAccess(TestContext.ManagedType, TestContext.ManagedMethod);
+        }
+
+        [TestMethod]
+        [RequiresApiAccess]
+        public void CustomerAddresses_GetCustomerAddresses_IntegrationTest()
+        {
+            var result = CrudHelpers.GetAll(() => IntegrationTestHelpers.ApiClient.CustomerAddresses.GetCustomerAddresses(1, 5));
+            Assert.IsNotNull(result);
+            Assert.IsNotNull(result.Data);
```

Comment:
Diese Assertions sind eigentlich doppelt -> sie werden ja schon in den CrudHelpers.GetAll aufgerufen

Reply:
Ok, wird entfernt.